src/js/control-bar/text-track-controls/chapters-button.js

```diff
@@ -124,7 +98,8 @@ class ChaptersButton extends TextTrackButton {
 
         const mi = new ChaptersTrackMenuItem(this.player_, {
           cue,
-          track: chaptersTrack
+          track: chaptersTrack,
+          selectable: true
```

Comment:
Okay, then we need to discuss the concept of "selected" some more. I see that chapter menu items will update as "selected" based on the timeline, but from an ARIA perspective, it's not strictly "selected". Again, think of it as a checkbox - if a checkbox is already checked, and you "click" on it, you don't expect the checkbox to stay checked. If it's a radio button you do, but then you don't expect it to have another effect. So this Chapters menu is a hybrid control, which is both a user control and a progress display. I understand what @chemoish  is trying to achieve visually, but it's not doing the correct thing logically, so just making it "selectable" isn't the correct approach. (Having said that, using the `controlText` to indicate `selected` to a screen reader was a hack too! Let's fix this "properly" before we go too far down the road to fix it at all).